README.rst

```diff
@@ -1833,51 +1831,40 @@ for configuring and running SQLAlchemy in a production system.
 
 .. code:: python
 
-    from graphql import parse
-    from graphql.utils.build_ast_schema import build_ast_schema
-    from sqlalchemy import MetaData, Table, Column, String, create_engine
-    from graphql_compiler.compiler.ir_lowering_sql.metadata import SqlMetadata
-    from graphql_compiler.schema.schema_info import make_sqlalchemy_schema_info
-    from graphql_compiler import graphql_to_sql
+    from sqlalchemy import MetaData, create_engine, dialects
```

Comment:
nit: Sort your imports so people can copy-paste this without lint errors.

Reply:
Depending on how the script is run the way you need to sort imports might be different.  
https://github.com/timothycrosley/isort#installing-isorts-for-your-preferred-text-editor

I sorted the imports assuming `graphql_compiler` is third party library because that is the way users are going to run it.